src/Parser/RestoDruid/Modules/Traits/EssenceOfNordrassil.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import calculateEffectiveHealing from 'Parser/Core/calculateEffectiveHealing';
+import SPELLS from 'common/SPELLS';
+
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+const ESSENCE_OF_NORDRASSIL_HEALING_INCREASE = 0.1;
```

Comment:
Shouldn't this be 0.05?

Reply:
Right, I even had it written down on the to-fix list but forgot about it. T_T

Reply:
Done